app/lib/core/components/PlayButton.tsx

```diff
@@ -6,7 +6,7 @@ const PlayButton = () => {
   return (
     <NavLink
       className="bg-jam-600 font-pixel hover:bg-jam-700 mt-6 flex w-fit items-center justify-baseline gap-4 rounded-full px-8 py-2 text-4xl font-bold text-white shadow-lg transition-transform duration-200 hover:scale-103"
-      to="/game"
+      to="/game/calculator"
```

Comment:
i'd leave this as /game. in the future I want to change /game to /levels. /levels should then just redirect to the first level imo